### PR TITLE
Internationalization: Step 1

### PR DESCRIPTION
**Pull request by liberavia**
This pull is related to issue https://github.com/lutris/lutris/issues/728

It contains marking any found string that seems to be played out in gui, so it is some sort of base for using gettext.

### CODE REVIEW

File: lutris/installer/commands.py [hunk 1]
```diff
@@ -173,16 +173,15 @@ def _on_input_menu_validated(self, widget, *args):
...
-            'message',
-            "Insert or mount game disc and click Autodetect or\n"
-            "use Browse if the disc is mounted on a non standard location."
-        )
-        message += (
-            "\n\nLutris is looking for a mounted disk drive or image \n"
-            "containing the following file or folder:\n"
-            "<i>%s</i>" % requires
-        )
+        message_action = _("Insert or mount game disc and click Autodetect or") + "\n"
```
Comment (TingPing) [inline lutris/installer/commands.py]: This shouldn't be split like this, it turns a normal paragraph into weird partial sentences.

File: lutris/runners/atari800.py [hunk 2]
```diff
@@ -26,14 +26,29 @@ class atari800(Runner):
...
         "5200_rom": ""
     }
+
+    def get_long_help_msg(self,option):
+        help_messages = {}
+
+        # main_file
+        help_messages['main_file'] = _("The game data, commonly called a ROM image.") + " \n"
+        help_messages['main_file'] += _("Supported rom formats: ATR, XFD, DCM, ATR.GZ, XFD.GZ")
+        help_messages['main_file'] += _("and PRO.")
```
Comment (TingPing) [inline lutris/runners/atari800.py]: Same comment about split line.

File: lutris/runners/atari800.py [hunk 3]
```diff
@@ -26,14 +26,29 @@ class atari800(Runner):
...
+
+        # bios_path
+        help_messages['bios_path'] += _("A folder containing the Atari 800 bios files.") + " \n"
+        help_messages['bios_path'] += _("They are provided by Lutris so you shouldn't have to") + " "
+        help_messages['bios_path'] += _("change this") + "\n"
```
Comment (TingPing) [inline lutris/runners/atari800.py]: Same.

File: lutris/gui/config_dialogs.py [hunk 4]
```diff
@@ -452,8 +452,9 @@ def get_config_id(self):
 class EditGameConfigDialog(Dialog, GameDialogCommon):
     """Game config edit dialog."""
     def __init__(self, parent, game, callback):
+        headline_game_config_dialog = _("Configure ") + game.name
         super(EditGameConfigDialog, self).__init__(
-            "Configure %s" % game.name,
+            headline_game_config_dialog,
```
Comment (TingPing) [inline lutris/gui/config_dialogs.py]: So I believe it is best to actually expose `%s` here as it allows RTL languages to be in the correct order. Same for a few others. It does have the downside that bad translations can cause runtime errors but there are tools to validate this.

**Comment by liberavia** [comment]
I changed those lines the way strycore suggested in https://github.com/lutris/lutris/issues/728#issuecomment-360007657

Could someone review this to make sure this is the way you want it to be implemented before I go over all files again?

File: lutris/gui/installgamedialog.py [hunk 5]
```diff
@@ -263,18 +264,18 @@ def prepare_install(self, script_index):
...
             self.set_path_chooser(self.on_target_changed, 'folder',
                                   default_path)
             self.non_empty_label = Gtk.Label()
-            self.non_empty_label.set_markup(
-                "<b>Warning!</b> The selected path "
-                "contains files, installation might not work properly."
-            )
+            label_markup = "<b>" + _("Warning!") + "</b>"
+            label_markup += " " + _("The selected path") + " "
+            label_markup += _("contains files, installation might not work properly.")
```
Comment (liberavia) [inline lutris/gui/installgamedialog.py]: Todo: Remove splitting of parts that belong together

File: lutris/gui/sync.py [hunk 6]
```diff
@@ -68,8 +68,9 @@ def __init__(self, parent=None):
         self.get_content_area().add(box_outer)
 
         description_label = Gtk.Label()
-        description_label.set_markup("You can import games from local game sources, \n"
-                                     "you can also choose to sync everytime Lutris starts")
+        label_markup = _("You can import games from local game sources,") + " \n"
+        label_markup += _("you can also choose to sync everytime Lutris starts")
+        description_label.set_markup(label_markup)
```
Comment (liberavia) [inline lutris/gui/sync.py]: Todo: line and meaning splitting

File: lutris/gui/uninstallgamedialog.py [hunk 7]
```diff
@@ -82,10 +82,12 @@ def on_apply_button_clicked(self, widget):
         if remove_contents and not hasattr(self.game.runner,
                                            'no_game_remove_warning'):
             game_dir = self.game.directory.replace('&', '&amp;')
+            question =  _("Are you sure you want to delete EVERYTHING under") + " \n<b>"
+            question += game_dir
+            question += "</b>?\n " + _("(This can't be undone)")
+
```
Comment (liberavia) [inline lutris/gui/uninstallgamedialog.py]: Using pattern instead of only concatenation

**Comment by TingPing** [comment]
> There is a problem with resolving the problem not having markup in translations and keeping textblocks together. Any suggestions how to deal with this?

If it comes down to it not splitting up sentences is more important than some markup being leaked IMO.

**Comment by strycore** [comment]
We are not splitting strings for i18n, this is my final design decision on the subject.

**Comment by TingPing** [comment]
Hmm, I'm not sure if using `{runner}` over `%s` is a great idea. It is bound to be translated incorrectly by somebody and also it bypasses the `gettext` tooling which is aware of `%s` style formatting.

**Comment by liberavia** [comment]
You mean I should do it this way?

```
mystring = _(
    "This %s message"
    "goes over"
    "several lines"
) % some_message_type
```

**Comment by strycore** [comment]
This is now too old and too out of sync to be rebased against master. A new PR has to be submitted.

**Pull request closed without merging**